dart/lib/src/client.dart

```diff
@@ -45,6 +51,11 @@ abstract class SentryClient {
   }) async {
     event = _processEvent(event, eventProcessors: options.eventProcessors);
 
+    // dropped by sampling or event processors
+    if (event == null) {
+      return Future.value(SentryId.empty());
```

Comment:
@rxlabz should we use `Future.value(SentryId.empty());` or just `SentryId.empty()`?
cus the NoOp Client and Hub return `Future.value(...)` but the real implementation seems to return the String.

Reply:
It returns a Future so it wouldn't compile without this.

Or mark the method `async` but without an `await` in the body that wouldn't make sense

Reply:
It does compile, that's why the question, i guess the "compiler" infers that automatically.